### DISCUSSION TITLE
Ammo vs Cannon

### DISCUSSION THREAD

**Discussion by vinkovsky** [pmndrs/use-cannon#181] [Q&A]
Hello! Thanks for the great library! Can you explain to me which is better to use Ammo or Cannon for the performance? About a year ago, I saw the official cannon examples on the three.js website, but now it's only ammo.

**Comment by vinkovsky** [comment]
Does the cannon support a soft body [like this](https://threejs.org/examples/?q=ammo#physics_ammo_volume)?

**Reply by stockhuman** [reply]
Hey @vinkovsky, the answer is no, or at least not without work-arounds that I've never explored. Cannon supports cloth and constraints, witch can ultimately achieve that, but not like Ammo. And indeed, cannon was for a while abandoned, so that makes sense.
